frontend/pages/add_team_members.vue

```diff
@@ -0,0 +1,202 @@
+            filled
+          />
+        </template>
+      </UButton>
+    </div>
+    <div class="ml-[10%] mr-[40%]">
+      <p class="ml-[3%] font-heading text-3xl font-medium">{{ roomCode }}</p>
+      <h1
+        class="mb-[5%] ml-[3%] font-heading text-6xl font-bold text-sc-orange"
+      >
+        {{ teamName }}
+      </h1>
+      <div
+        class="mb-4 flex max-h-[43vh] flex-grow-0 flex-col justify-center rounded-lg border-2 border-sc-black-400 bg-sc-white p-7 drop-shadow-sc-48025"
+      >
+        <div class="flex overflow-y-scroll">
+          <table class="w-full">
+            <tbody>
+              <tr
+                v-for="member in teamMembers"
+                :key="member.id"
+                class="flex justify-between border-b-2 border-sc-black"
+              >
+                <td class="flex-grow p-2 text-3xl">{{ member.name }}</td>
+                <td class="flex items-center">
+                  <select
+                    class="mr-4 rounded-md p-1"
+                    v-model="member.role"
+                    @change="updateRole(member.id, $event.target.value)"
+                  >
+                    <option
+                      v-for="role in roles"
+                      :key="role.label"
+                      :value="role.label"
+                      :disabled="isRoleTaken(role.label)"
+                    >
+                      {{ role.label }}
+                    </option>
+                  </select>
+                  <UButton
+                    @click="deleteMember(member.id)"
+                    icon="mdi:trash-can-outline"
+                    class="size-10 justify-center hover:bg-sc-black-100"
+                    :padded="false"
+                    variant="ghost"
+                    color="sc-orange"
+                    :ui="{
+                      rounded: 'rounded-full',
+                      icon: { size: { sm: 'size-6' } },
+                    }"
+                  >
+                  </UButton>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <!-- @submit.prevent is a workaround so the page isn't reloaded till the backend exists-->
+        <form @submit.prevent>
+          <input
+            class="mt-7 w-full rounded-lg bg-sc-yellow/70 p-1 text-center text-3xl font-bold drop-shadow-sc-48025 placeholder:text-sc-black focus:placeholder:text-transparent"
+            :placeholder="$t('join-room.add-team-member')"
+            v-model="newMember"
+          />
+          <button @click="addMember()" />
+        </form>
+      </div>
+      <div class="flex w-full justify-end">
+        <button
+          :disabled="!isReady()"
+          class="w-72 rounded-lg py-6 text-center text-4xl font-bold drop-shadow-sc-48025"
+          :class="
+            isReady()
+              ? 'bg-sc-green text-sc-black hover:bg-sc-green-400'
+              : 'bg-sc-black-400 text-sc-white'
+          "
```

Comment:
dazu gibts eine bessere Syntax 
https://vuejs.org/guide/essentials/class-and-style